src/game/maps/chemlab/storage_room.gd

```diff
@@ -0,0 +1,8 @@
+extends Node2D
+
+func _ready():
+	if not Connections.isServer():
+		return
+	for i in range(0, 2):
```

Comment:
I am of the mind that the loop variables should be descriptive of what is being looped. In this case, it seems to be setting the x and y positions, so why not use x and y as variables here?

Mainly nitpicking because I can't actually see any major problems with this PR so far lol.